docs/build_process.rst

```diff
@@ -64,28 +53,30 @@ variable::
 Example
 ~~~~~~~
 
-Here is an example Python session demonstrating this interface::
+Here is an example Python session demonstrating this interface for streaming::
 
   >>> server = OSBS(...)
-  >>> logs = server.get_build_logs(...)
-  >>> [(item.platform, item.line) for item in logs]
-  [(None, '2017-06-23 17:18:41,791 platform:- - atomic_reactor.foo - DEBUG - this is from the pipeline task'),
-   ('x86_64', '2017-06-23 17:18:41,400 atomic_reactor.foo - DEBUG - this is from a build'),
-   ('x86_64', 'continuation line')]
+  >>> logs = server.get_build_logs(follow=True, wait=True)
+  >>> [(task_run_name, log_line) for task_run_name, log_line in logs]
+  [('task_run', '2017-06-23 17:18:41,791 platform:- - atomic_reactor.foo - DEBUG - this is from the pipeline task'),
+   ('task_run_x86-64', '2017-06-23 17:18:41,400 atomic_reactor.foo - DEBUG - this is from a build'),
+   ('task_run_x86-64', 'continuation line')]
```

Comment:
Could we have slightly more real examples? Maybe `prebuild`, `build-x86-64`?

Reply:
added real task names. no more underscores either